### DISCUSSION TITLE
Override node-linker with environment variable?

### DISCUSSION THREAD

**Discussion by raygesualdo** [pnpm/pnpm#4396] [Q&A]
Hey all! 👋  Needing to configure the `node-linker` option for a single command separately from how the repo will normally be configured. Is there a way to pass that option in as an environment variable?

**Comment by styfle** [comment]
I'm also wondering the same thing since the documentation mentions environment variables but doesn't give an example:

> pnpm gets its configuration from the command line, environment variables, and .npmrc files.

https://pnpm.io/npmrc
